elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/compile/ElideDynamicEntityCompiler.java

```diff
@@ -201,4 +238,38 @@ public ClassLoader getClassLoader() {
                .map(Class::getName)
                .collect(Collectors.toList());
     }
+
+    /**
+     * Prepare Import statement.
+     * @param pkgName package name.
+     * @param className class name.
+     * @return complete import statement.
+     */
+    private static String prepareImport(String pkgName, String className) {
+        return IMPORT + pkgName + DOT + className + SEMI_COLON;
+    }
+
+    /**
+     * Get Class Name for provided (modelName, modelVersion).
+     * @param modelName model name.
+     * @param modelVersion model version.
+     * @param defaultValue default value.
+     * @return class name for provided (modelName, modelVersion) if available else default value.
+     */
+    public static String getStaticModelClassName(String modelName, String modelVersion, String defaultValue) {
+        ModelMapValue value = STATIC_MODEL_DETAILS.get(new ModelMapKey(modelName, modelVersion));
```

Comment:
We should use these lookups to validate the joins as well (so we don't get a cryptic error here if they put down a class that doesn't exist).

Reply:
Added validation on Join's `to` value